programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,12 @@
+function solution(num) {
+    var number
+    if (num %2 ===0){
+        number = "Even";
+    }
+    else{
+        number = "Odd";
+    }
+    return number;
+}
```

Comment:
if문 조건을 통과했을시 바로 return 해주는것이 더 깔끔한 코드가 됩니다.
if문을 통과하지 못한다면 홀수라는 의미가 되니 else문이 굳이 필요 없습니다.

```suggestion
    if (num %2 === 0) {
        return "Even";
    }
    return "Odd";
}
```